app/resources/css-default.template

```diff
@@ -208,13 +208,19 @@ input:focus {
   border-bottom: 2px solid #3b78e7;
 }
 
+/* Show disabled input states with dotted line */
 input:not([type]):disabled {
```

Comment:
What is the meaning of "not([type])"? If you follow my suggestion above, then maybe you should also apply this for limited types of inputs.